plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -3317,7 +3318,7 @@ public boolean cleanupRules() {
         if (!_canBridgeFirewall) {
             return false;
         }
-        final Script cmd = new Script(_securityGroupPath, _timeout, s_logger);
+        final Script cmd = new Script(_securityGroupPath, _timeout.getMillis(), s_logger);
```

Comment:
Scripts is an external package, hope they will upgrade to Duration after Java 1.8.